christfellowshipchurch/web-app-v2
# Removed Christmas Banner from Header

### About
This PR removes the `ChristmasBanner` component from the top of headers in the website since it no longer has use for us.

### Test Instructions
- Go to external home page to see that it is removed
- Go to any other page to see the default header to see that it is removed there as well

### Screenshots
![image](https://user-images.githubusercontent.com/46049974/147255634-70947f63-f62c-4d60-83fc-29a64f148c69.png)

### Closes Tickets
[CFDP-1906]


[CFDP-1906]: https://christfellowshipchurch.atlassian.net/browse/CFDP-1906?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
